Makefile

```diff
@@ -40,34 +43,34 @@ test_provider::
 	cd provider/pkg && go test -short -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM} ./...
 
 dotnet_sdk:: DOTNET_VERSION := $(shell pulumictl get version --language dotnet)
-dotnet_sdk::
+dotnet_sdk::	.pulumi/bin/pulumi
 	rm -rf sdk/dotnet
-	pulumi package gen-sdk --language dotnet $(SCHEMA_FILE)
+	.pulumi/bin/pulumi package gen-sdk --language dotnet $(SCHEMA_FILE)
```

Comment:
If we are adding this to `PATH`, we shouldn't need to prefix `pulumi`.

Reply:
See https://github.com/pulumi/pulumi-command/pull/266